source/process/release-scorecard-definitions.rst

```diff
@@ -0,0 +1,189 @@
+|                                          | “=Release Day-Created”.                        |   
++------------------------------------------+------------------------------------------------+
+| Mean time from P2 bug report to delivery | With a new or existing Jira filter, with:      |
+|                                          |                                                |
+|                                          | 1. Project = Mattermost                        |
+|                                          | 2. Fix Versions = Latest released version      |
+|                                          | 3. Issue Type = Bug                            |
+|                                          | 4. Label = P2                                  |
+|                                          | 5. Created Date = 17th of the previous month   |
+|                                          | 6. Release Date = 16th of the current month    |
+|                                          |                                                |
+|                                          | Copy a list of Jira tickets with the above     |
+|                                          | information and paste them into a spreadsheet. | 
+|                                          | Calculate the average by using a formula       |
+|                                          | “=Release Day-Created”.                        |
++------------------------------------------+------------------------------------------------+
+| Mean time from P3 bug report to delivery | With a new or existing Jira filter, with:      |
+|                                          |                                                |
+|                                          | 1. Project = Mattermost                        |
+|                                          | 2. Fix Versions = Latest released version      |
+|                                          | 3. Issue Type = Bug                            |
+|                                          | 4. Label = P3                                  |
+|                                          | 5. Created Date = 17th of the previous month   |
+|                                          | 6. Release Date = 16th of the current month    |
+|                                          |                                                |
+|                                          | Copy a list of Jira tickets with the above     |
+|                                          | information and paste them into a spreadsheet. | 
+|                                          | Calculate the average by using a formula       |
+|                                          | “=Release Day-Created”.                        |
++------------------------------------------+------------------------------------------------+
+| Number of P1 bugs reported in production |                                                |
```

Comment:
Propose adding a note that bug severity guidelines are in progress, after which a Jira query will be built